deploy/jenkins-ci/Makefile

```diff
@@ -20,10 +20,22 @@ ifeq ($(RELEASE_TYPE),minor)
   # For minor, remove the -SNAPSHOT suffix from the Makefile
   BACKEND_VERSION_TO_RELEASE ?= $(shell semver bump release $(BACKEND_VERSION))
 endif
+
+ifeq ($(findstring snapshot,$(RELEASE_TYPE)),snapshot)
+  RELEASE_TYPE_IS_SNAPSHOT_OR_EDGE = y
+else ifeq ($(RELEASE_TYPE),edge)
+  RELEASE_TYPE_IS_SNAPSHOT_OR_EDGE = y
+else
+  RELEASE_TYPE_IS_SNAPSHOT_OR_EDGE = n
+endif
+
 ifeq ($(findstring snapshot,$(RELEASE_TYPE)),snapshot)
   # For end-of-week snapshot release, substitute the -SNAPSHOT prefix in the Makefile with
   # the value of $(RELEASE_TYPE) -- assuming value will be snapshot.{X}
   BACKEND_VERSION_TO_RELEASE ?= $(shell semver bump prerel $(RELEASE_TYPE) $(BACKEND_VERSION))
+else ifeq ($(RELEASE_TYPE),edge)
+  # Use 'latest' for edge releases
+  BACKEND_VERSION_TO_RELEASE ?= latest
```

Comment:
I thought it was OK, but I laughed after seeing that I got this:
![image](https://user-images.githubusercontent.com/23639005/52675041-1372e580-2eeb-11e9-9353-d1c91de2e38f.png)

Haha... still not right :)

Reply:
Ha!

I didn't see that one coming.

Reply:
Ok, Removed the `v`